pkg/operation/botanist/secrets.go

```diff
@@ -21,6 +21,8 @@ import (
 	"net"
 	"os/exec"
 
+	"github.com/gardener/gardener/extensions/pkg/util"
```

Comment:
Can you avoid using something from the `extensions` library? It might result in future dependency cycles.
Rather move the function in question to some other package in `pkg`